static/js/tools/map/context.ts

```diff
@@ -58,7 +58,7 @@ export interface StatVarInfoWrapper {
 // Information relating to the places to plot
 export interface PlaceInfo {
   // Place that encloses the places to plot
-  enclosingPlace: NamedPlace;
+  enclosingPlace: Node;
```

Comment:
Just wondering, why the change from NamedPlace to Node? I feel like Node is a very generic name that could be confusing

Reply:
was trying to unify the type, but you are right, NamedPlace is better here. I added type alias with NamedNode = NamedPlace now and reverted these changes